### PR TITLE
Document LEVEL_TYPE: BIOMESOP

### PR DESCRIPTION
**Pull request by MCOfficer**
This can only be found by digging through issues (#156), so it would make sense to mention it in the README.

### CODE REVIEW

**Comment by MCOfficer** [comment]
Turns out that BoP post-1.15 no longer needs `BIOMESOP` but `BIOMESOPLENTY`. Would you like me to add another exception, or just remove the check entirely?

**Comment by itzg** [comment]
That PR you linked it so old that the validation of `LEVEL_TYPE` was long since removed. It just converts to upper case and stores whatever value the use gives:

https://github.com/itzg/docker-minecraft-server/blob/3de2bf88df93a0ef8ef41830dafb22ca6b45962e/start-finalSetupServerProperties#L76

File: README.md [hunk 1]
```diff
@@ -967,6 +967,7 @@ be configured by setting `LEVEL_TYPE` to an expected type, such as
 - AMPLIFIED
 - CUSTOMIZED
 - BUFFET
+- BIOMESOP (if Biomes O' Plenty is installed)
```
Comment (itzg) [inline README.md:970]: Since this list only serves as examples now, you could clarify the `BIOMESOPLENTY` you found for 1.15+

**Comment by MCOfficer** [comment]
> That PR you linked it so old that the validation of `LEVEL_TYPE` was long since removed. It just converts to upper case and stores whatever value the use gives:

Wow, I really should've checked the master branch :laughing: 

Other than adding BIOMESOPLENTY, i also took the liberty to reword it to "for example", because i really thought that list was something akin to an enum.

**Comment by itzg** [comment]
> Other than adding BIOMESOPLENTY, i also took the liberty to reword it to "for example", because i really thought that list was something akin to an enum.

Thanks. Yes, in your defense the section was indeed still worded as if it was an enum of allowed values.

**Review by itzg — APPROVED** [review]
Thanks for updating that section.
